pkg/operator/ceph/object/livenessprobe.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2020 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package object allows a ceph config file to be stored in Kubernetes and mounted as volumes into
+// Ceph daemon containers.
+package object
+
+import (
+	cephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
+	v1 "k8s.io/api/core/v1"
+)
+
+// configureLivenessProbe returns the desired liveness probe for a given daemon
+func configureLivenessProbe(container v1.Container, healthCheck cephv1.BucketHealthCheckSpec) v1.Container {
```

Comment:
A more common pattern is to pass a pointer, then you don't need to return the same object. I see you followed the pattern of the other liveness probe, but we can fix that separately.
```suggestion
func configureLivenessProbe(container *v1.Container, healthCheck cephv1.BucketHealthCheckSpec) {
```

Reply:
Done.